.gitlab-ci.yml

```diff
@@ -56,6 +56,14 @@ flake8-diff:
     - apk add --update --no-cache git
     - git diff -U0 origin/master | flake8 --verbose --diff
 
+shellcheck:
+  stage: build
+  image: koalaman/shellcheck-alpine:v0.7.1
+  before_script:
+    - apk add --update --no-cache git
+  script:
+    - git ls-files -z '*.sh' '*.sh.in'  | xargs -0 shellcheck -x
```

Comment:
What happened to the -P option?

Reply:
I haven't found description of this option in man page of shellcheck version 0.4.3 (which i have on Ubuntu).
Double check using more fresh version (0.7.0) let me found it. So i brought it back.